raster_cutter.py

```diff
@@ -411,17 +467,30 @@ def generate_lexocad_files(directoryUrl):
     height = abs(src_height * float(lines[3]))
     xMinimum = float(lines[4])
     yMinimum = float(lines[5]) - height
-    with open(directoryUrl + "l", 'w') as f:
+    with open(directoryUrl + 'l', 'w') as f:
         f.write(
-            str(xMinimum) + "\n" +
-            str(yMinimum) + "\n" +
-            str(float(width)) + "\n" +
-            str(float(height)) + "\n" +
-            "\n" +
-            "# cadwork swisstopo" + "\n" +
-            "# " + str(xMinimum) + " " + str(yMinimum) + "\n" +
-            "# " + str(width) + " " + str(height) + "\n" +
-            "# projection: EPSG:2056 - CH1903+ / LV95"
+            str(xMinimum)
+            + '\n'
+            + str(yMinimum)
+            + '\n'
+            + str(float(width))
+            + '\n'
+            + str(float(height))
+            + '\n'
+            + '\n'
+            + '# cadwork swisstopo'
+            + '\n'
+            + '# '
+            + str(xMinimum)
+            + ' '
+            + str(yMinimum)
+            + '\n'
+            + '# '
+            + str(width)
+            + ' '
+            + str(height)
+            + '\n'
+            + '# projection: EPSG:2056 - CH1903+ / LV95'
```

Comment:
@hixi Was this auto-format? I think the previous version makes more sense because each line is a line in the generated doc.